### ISSUE TITLE
Angle field fails to take ephemeral focus

### ISSUE THREAD

**Issue by rachel-fenichel** [rachel-fenichel/blockly-samples#2515]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Component

`field-angle`

### Description

Error in console: attempted to take ephemeral focus when it's already held.

New issue with v12.

### Reproduction steps

Link against Blockly v12.0.0-beta.5 and open the plugin test page.

Click an angle field to open the editor.

### Stack trace

```shell

```

### Screenshots

Same issue as the slider issue reported in #2514--note that this screenshot is from the slider, but the details still apply.

![Image](https://github.com/user-attachments/assets/5f6b0514-b8ff-426b-97a1-66a09c322222)

**Comment by BenHenning** [comment]
This has the same issue and fix as https://github.com/google/blockly-samples/issues/2514#issuecomment-2881539117.

**Comment by BenHenning** [comment]
Fixed in https://github.com/google/blockly-samples/pull/2521.
